.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: java
+
+jdk:
+  - oraclejdk8
+  - openjdk8
+
+script: "mvn cobertura:cobertura"
```

Comment:
I'm not sure we really want to generate test coverage numbers, we just want to run the tests.  Since this project is just samples of how to use AndHow and how to write tests, we don't really care what percentage test coverage we have.

Reply:
Maybe you have a different idea here and see a need for the code coverage report.  If so, please discuss on the developer discussion list.

Reply:
True, @eeverman . Guess I didn't really think this through 🤦‍♂️ 

Guess `mvn package` should suffice. Or even `mvn test`

Reply:
going with `mvn clean package` to begin with